addons/api/addon/models/worker.js

```diff
@@ -34,14 +34,18 @@ export default class WorkerModel extends GeneratedWorkerModel {
   }
 
   /**
-   * Returns the number of canonical tags present on the worker.
+   * Returns the number of tags present on the worker.
    * @type {number}
    */
   get tagCount() {
     if (!this.canonical_tags) {
       return 0;
     }
```

Comment:
Thanks for spotting this bug!

Looking at this now, we need to change this check here. We updated the serializer to always set `canonical`, `config`, and `api` tags to an empty object if they are not present. Maybe something like this is what we need?
```suggestion
    if (
      !Object.keys(this.config_tags).length &&
      !Object.keys(this.api_tags).length
    ) {
      return 0;
    }
```

Reply:
ah yes great catch, thank you!